test/sanity_check.sh

```diff
@@ -6,10 +6,17 @@ set -e
 
 whitely echo Ping each host from the other
 
+function check_ping() {
```

Comment:
Do you think this is clear enough?

```
# We wrap ping and echo in a function as we want the below parallel for loop
# to exit early in case of a failed ping, and it cannot be done concisely,
# e.g. using a one-liner, without losing the status code for ping.
```